ouster_client/include/ouster/client.h

```diff
@@ -98,6 +100,7 @@ std::shared_ptr<client> init_client(
     const std::string& hostname, const std::string& udp_dest_host,
     lidar_mode ld_mode = MODE_UNSPEC, timestamp_mode ts_mode = TIME_FROM_UNSPEC,
     int lidar_port = 0, int imu_port = 0,
+    OperatingMode operating_mode = OPERATING_NORMAL,
```

Comment:
If you move the new argument to be after persist_config this would be a non-breaking change